plugins/foreman_discovery/4.0/index.md

```diff
@@ -446,6 +446,11 @@ that, set the value of this setting to name of a fact reported. To hide the
 new column, set to a blank value. To show multiple columns, separate fact
 names by comma.
 
+The new discovery view uses regular expressions to decide which category each fact belongs to,
```

Comment:
I'd also suggest not referring to something as "new" in documentation, as it's presumably going to live on.